src/v2/Apps/Order/OrderApp.tsx

```diff
@@ -135,18 +135,22 @@ class OrderApp extends React.Component<OrderAppProps, {}> {
     }
 
     const stripePromise = loadStripe(sd.STRIPE_PUBLISHABLE_KEY)
+
+    const isModal = this.props.match?.location.query.isModal ? true : false
+
     return (
       <SystemContextConsumer>
         {({ isEigen, mediator }) => {
           // @ts-expect-error STRICT_NULL_CHECK
           this.mediator = mediator
+
           return (
             <Box>
               {/* FIXME: remove once we refactor out legacy backbone code.
-                  Add place to attach legacy flash message, used in legacy inquiry flow
-              */}
+                    Add place to attach legacy flash message, used in legacy inquiry flow
+                */}
               <div id="main-layout-flash" />
-              <MinimalNavBar to={artworkHref}>
+              <MinimalNavBar to={artworkHref} isBlank={isModal}>
```

Comment:
What does `isBlank` refer to here? What is the use case for `isBlank`?

Reply:
`isBlank` hides the logo that is usually present in `MinimalNavBar`. `MinimalNavBar` suppresses the default nav bar which we want, we just don't want any content there at all